### DISCUSSION TITLE
How to prevent shrinking or resizing of app widgets while the keyboard is active

### DISCUSSION THREAD

**Discussion by fluttedevs** [community/community#29913] [Mobile] [answered]
How to prevent shrinking or resizing of app widgets while the keyboard is active

**Comment by surajadkhari** [comment, marked as answer]
Add "resizeToAvoiceBottomInset" to "false" to prevent widget resizing on the keyboard popup.

Scaffold(
  resizeToAvoidBottomInset: false
)

**Reply by fluttedevs** [reply]
Thank you
